.env.example

```
@@ -1,35 +1,40 @@
-# DJANGO SETTINGS
+# DOCKER-COMPOSE SETTINGS
 # ---------------------------------------------------------------------------
-# Django secret key (MUST be changed for production)
-# Generate a secure secret key with: python -c "from django.core.management.utils import get_random_secret_key; print(get_random_secret_key())"
-SECRET_KEY=your-secret-key-here
\ No newline at end of file
+# These variables are used by docker-compose.yml to configure services.
+# They are not directly used by Django, but are related to the database setup.
+DB_NAME=django_db
+DB_USER=django_user
+DB_PASSWORD=django_password
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_USER key

(UnorderedKey)